### PR TITLE
Backport changelog fixes

### PR DESCRIPTION
**Pull request by rsgowman**

### CODE REVIEW

File: firebase-firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
-  are not public. This affects internal APIs that were previously obfuscated
-  and are not mentioned in our documentation.
+  are not public. This affects internal APIs that were previously obfuscated and
+  are not mentioned in our documentation.
 - [changed] Improved error messages for certain Number types that are not
   supported by our serialization layer (#272).
 
+# 18.1.0
+- [changed] Internal changes to ensure functionality alignment with other SDK
+  releases.
```
Comment (mikelehen) [inline firebase-firestore/CHANGELOG.md]: I thought we only used this message when we had no "real" changes to list, but apparently (below) we do in this case...?

I don't care strongly either way though.

Comment (wilhuff) [reply firebase-firestore/CHANGELOG.md]: It's in the public release notes, so might as well leave it. https://firebase.google.com/support/release-notes/android

Comment (rsgowman) [reply firebase-firestore/CHANGELOG.md]: Yeah, I asked about that too. The decision at the time was to put it in all sdks, even ones like ours that didn't *really* need it.

**Comment by mikelehen** [comment]
It seems like connected-check-changed is stuck or something (it's been going for an hour and the log is empty).  Since this PR has no code changes and I'm impatient to merge it into https://github.com/firebase/firebase-android-sdk/pull/340, I'm just going to merge it.

**Pull request merged**